apps/livreGUI/transferFunctionEditor/ColorMapWidget.h

```diff
@@ -44,10 +44,11 @@ class ColorMapWidget : public TFWidget
      */
     enum ShadeType
     {
-        RED_SHADE,
-        GREEN_SHADE,
-        BLUE_SHADE,
-        ARGB_SHADE
+        RED_SHADE   = 0,
+        GREEN_SHADE = 1,
+        BLUE_SHADE  = 2,
+        ALPHA_SHADE = 3,
+        SHADE_COUNT = 4
```

Comment:
Oh, you introduced SHADE_COUNT here which is a very good idea! Use it to replace the hardcoded '4' in the for-loops where no range-based for loop can be used!